lib/streamlit/state/session_state.py

```diff
@@ -383,7 +383,12 @@ def get_session_state() -> SessionState:
     directly. Instead, SessionState objects should be accessed via
     st.session_state.
     """
-    return _get_current_session().session_state
```

Comment:
nit: should get rid of `_get_current_session` above since I don't believe it's used anywhere else

Also, I'm surprised that there aren't a few other tests failing as I thought that we had a few tests that verified that an error was thrown if `st.session_state` was accessed when no session exists, but I may be misremembering things (or the test may have already been removed).

Reply:
They probably existed in the old `session_state_tests.py`, which was removed in the unification PR